lib/sidekiq/antidote/inhibitor.rb

```diff
@@ -38,14 +38,34 @@ def match?(job_record)
       end
 
       def to_s
-        "#{treatment} #{class_qualifier}"
+        "#{treatment} #{class_qualifier}" + (treatment == "suspend" ? ": #{id}" : "")
```

Comment:
We don't need to add the id of inhibitor here - it's the implementation detail.
```suggestion
        "#{treatment} #{class_qualifier}"
```